modules/database/sqlite/db.copy

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Plugin Name: Performance Lab - SQLite module.
+ * Plugin URI: https://github.com/WordPress/performance
+ * Description: Performance plugin from the WordPress Performance Team, which is a collection of standalone performance modules.
+ * Requires at least: 6.0
+ * Requires PHP: 5.6
+ * Version: 1.7.0
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: performance-lab
+ *
+ * This file is auto-generated and copied from the performance-lab plugin.
+ * Please don't edit this file directly.
+ *
+ * @package performance-lab
+ */
+
+// Bail early if the SQLite implementation was not located in the performance-lab plugin.
+if ( ! file_exists( '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php' ) ) {
+	return;
+}
+
+// Define SQLite constant.
+if ( ! defined( 'DATABASE_TYPE' ) ) {
+	define( 'DATABASE_TYPE', 'sqlite' );
+}
+
+// Require the implementation from the performance-lab plugin.
+require_once '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php';
+
+// Activate the performance-lab plugin if it is not already activated.
+add_action(
+	'current_screen',
+	function() {
+		if ( ! defined( 'PERFLAB_VERSION' ) ) {
+			if ( ! function_exists( 'activate_plugin' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/plugin.php';
+			}
+			if ( is_plugin_inactive( '{PERFLAB_PLUGIN}' ) ) {
+				activate_plugin( '{PERFLAB_PLUGIN}' );
+			}
+		}
```

Comment:
Not sure how I feel about automatically activating the plugin. I get the idea why, but I think it would be better to render an admin notice instead informing the admin about that and giving them a link to `plugins.php` to activate the plugin.

While it's not a desirable situation that this drop-in is loaded when the plugin isn't active, I don't think it technically causes any harm, does it? It would only be _confusing_. So I think a notice here is more appropriate than automatically activating the plugin.

Imagine there is _any_ problem in the deactivation logic for example; it would lead to this plugin being activated over and over again. That's why I think leaving it to the admin user is safer.

Reply:
When I think about this, my mind goes into a loop.. 😅 
I suppose it depends on what we want to do when the plugin is not activated.

Do we want to add a check in the `wp-content/db.php` file, and not load the SQLite db when the plugin is inactive?
* If we do, then there's an infinite loop in the logic which will prevent SQLite from ever being activated: I activate SQLite from my MySQL site, so a new db is created, but the plugin is not active by default, which means that SQLite won't be loaded and therefore the user can't ever activate the plugin on that db since their site falls-back to the MySQL db
* If we don't, then we're only just adding an extra step for users? I'm fine with that, but I don't really see the point... If you think it's better though, I'd happily make that change!

Reply:
@aristath I see what you mean; however I'm still a bit concerned about having unconditional logic that would always try to (re-)activate the plugin if it's not active.

This again leads me to point out the approach of installing the database right away to make the user experience smoother. If we do that, we would also be able to immediately "copy over" a few extra options e.g. the `active_plugins` one to ensure that Performance Lab remains active.

I spent some time working on that today and actually got to a working solution, which I'll commit here shortly.